pam/integration-tests/helpers_test.go

```diff
@@ -260,6 +265,8 @@ func artifactsPath(t *testing.T) string {
 	authdArtifactsDirSync.Do(func() {
 		defer func() { t.Logf("Saving test artifacts at %s", authdArtifactsDir) }()
 
+		authdArtifactsAlwaysSave = os.Getenv("AUTHD_TEST_ARTIFACTS_ALWAYS_SAVE") != ""
```

Comment:
We have some occurrences of `AUTHD_TEST_*` env vars, but most use `AUTHD_TESTS_*`. Let's consistently use one prefix.

Also, I would prefer "ALWAYS_SAVE_ARTIFACTS" after the prefix for readability.

```suggestion
		authdArtifactsAlwaysSave = os.Getenv("AUTHD_TESTS_ALWAYS_SAVE_ARTIFACTS") != ""
```

Reply:
Oh that was the name I also gave initially, then I changed to keep it consistent with the other artifacts var, but I'm fine either way

Reply:
E.g. for consistency with `AUTHD_TEST_ARTIFACTS_PATH`. Want me to change both?

Reply:
It's different, in this case using TESTS is fine, while when the variable is related to a specific test only it should be TEST.

I'll check the inconsistencies though